drivers/SmartThings/matter-lock/profiles/base-lock.yml

```diff
@@ -10,7 +10,7 @@ components:
         enabledValues:
         - locked
         - unlocked
-        - unknown
+        - not fully locked
```

Comment:
Is the expectation now that `unknown` events will just be ignored and not result in a change in the attribute state?

Reply:
It's more that the devices will no longer generate the `unknown` event

Reply:
Oh I see, this is because of the change you had made here: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1595

Thanks for the clarification!